core/BUILD

```diff
@@ -39,6 +39,7 @@ cc_library(
         "@com_google_absl//absl/synchronization",
         "@com_google_absl//absl/types:span",
         "@rang",
+        "@spdlog",
```

Comment:
WDYT about just unconditionally adding `#include "spdlog/fmt/ranges.h"` to `common/common.h`?  Then we don't need the extra `#include`s everywhere, and we don't need to declare the extra dependency on `@spdlog`.